assets/component-rating.css

```diff
@@ -14,6 +14,7 @@
 }
 
 .rating-star {
+  --color-rating-star: rgb(var(--color-foreground));
```

Comment:
I could see the loss of accent color for rating stars being one of the more noticeable cases. So I've at least captured the color in a variable up at the .rating-star scope so would be easy to change via custom css. Even though the color gets used in a gradient style within a pseudo element.

`.rating-star { --color-rating-star: #000; }`

Reply:
We don't. So quite I'm happy to simplify if we don't think it's valuable. But I'm basically just making it easier to revert the visual change without having to dissect the line below and determine for yourself that only the first instance of color-foreground is the one you need to change to get your color back.

`background: linear-gradient(90deg, var(--color-foreground) var(--percent), rgba(var(--color-foreground), 0.15) var(--percent));`